source/ocsp-support/ocsp-support.rst

```diff
@@ -194,9 +194,21 @@ This boolean option determines whether a MongoClient should refrain from
 reaching out to an OCSP endpoint i.e.  whether non-stapled OCSP should
 be disabled.  When set to true, a driver MUST NOT reach out to OCSP
 endpoints. When set to false, a driver MUST reach out to OCSP
-endpoints if needed (as described in 
+endpoints if needed (as described in
 `Specification: Suggested OCSP Behavior <ocsp-support.rst#id1>`__).
-This option MUST default to false.
+This option MUST default to false except in the case that a driver' TLS
+library exhibits hard-fail behavior (i.e. the driver fails the
+`"Soft Fail Test
+"<https://github.com/mongodb/specifications/tree/master/source/ocsp-support/tests#integration-tests-permutations-to-be-tested>`__
```

Comment:
I think the double quote character should be adjacent to "Test" on the previous line. As-is, you'll get a space between "Test" and the closing quote. You probably do want a space between the link text and the `<` character as well.

Reply:
Fixed. I currently have a new line between the link test and the <, which appears to work when viewing it on Github.